qa/tasks/ceph_manager.py

```diff
@@ -1373,7 +1373,9 @@ def raw_cluster_cmd_result(self, *args, **kwargs):
         kwargs['args'], kwargs['check_status'] = args, False
         return self.run_cluster_cmd(**kwargs).exitstatus
 
-    def run_ceph_w(self, watch_channel=None):
+    # XXX: setting shell to True by default is not good because killing the
+    # process ceph -w and running stdin.wait() on it leads to a crash.
```

Comment:
```suggestion
    # process ceph -w and running stdin.wait() on it leads to a hang.
```

?

Reply:
IIRC, the actual issue was that the shell gets killed but not `ceph -w`. Please be more descriptive about the problem.

Reply:
On looking now, I find this note redundant. The one is here is enough - https://github.com/ceph/ceph/pull/38443/files#diff-cf48a01154da3979f620dc03f242d20fa7e554b872bf7f6fbc982ee6bf58b37fR109.

Reply:
Added details to a tracker ticket and added the link to the tracker ticket to the note instead.